hazelcast/src/main/java/com/hazelcast/config/MemoryTierConfig.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config;
+
+import com.hazelcast.internal.config.ConfigDataSerializerHook;
+import com.hazelcast.memory.MemorySize;
+import com.hazelcast.memory.MemoryUnit;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * Memory tier configuration of Tiered-Store.
+ *
+ * @since 5.1
+ */
+public class MemoryTierConfig implements IdentifiedDataSerializable {
+
+    /**
+     * Default capacity. It is 256 MB.
+     */
+    public static final MemorySize DEFAULT_CAPACITY = new MemorySize(256, MemoryUnit.MEGABYTES);
+
+    private MemorySize capacity = DEFAULT_CAPACITY;
+
+    public MemoryTierConfig() {
+
+    }
+
+    public MemoryTierConfig(MemoryTierConfig memoryTierConfig) {
+        capacity = memoryTierConfig.getCapacity();
+    }
+
+    /**
+     * Returns the capacity of this memory tier.
+     *
+     * @return memory tier capacity.
+     */
+    public MemorySize getCapacity() {
+        return capacity;
+    }
+
+    /**
+     * Sets the capacity of this memory tier.
+     *
+     * @param capacity capacity.
+     * @return this MemoryTierConfig
+     */
+    public MemoryTierConfig setCapacity(MemorySize capacity) {
```

Comment:
I think we can also reflect this change to xml and yaml config.
Currently we have `native-memory` like this, you can copy same types to xsd and yaml.
```
native-memory:
    size:
      unit: GIGABYTES
      value: 100                          
```

Reply:
This is on the xsd and yaml side as well. So capacity can be configured as `capacity: 2048 MB`. I thought of doing it like in `native-memory`, but putting value and unit separately doent seem very nice

Reply:
Will not insist on this, i like consistency between similar elements all over the config. For xml, it will be easy to have separate parts since it suggest values but for yaml it may not seem very nice, yes.